integration.sh

```diff
@@ -8,14 +8,42 @@ stop_server() {
 	curl http://127.0.0.1:8000/STOP
 }
 
-integration_test_upload_diff() {
-	fastlane integration_test_upload_dif
-}
+start_server &
 
-if ! (start_server & integration_test_upload_diff) ; then
+if ! (fastlane integration_test_upload_dif) ; then
 	stop_server
   	exit 1
-else 
+fi
+
+if ! (fastlane integration_test_create_release) ; then
 	stop_server
-	exit 0
+  	exit 1
```

Comment:
We seem to be mixing tabs and spaces. What about going all-in on spaces? (yeah I'm a spaces guy)